MovieAppPetProject/MovieAppPetProject/GetMovieDetailsByID/GetMovieDetails-ViewController.swift

```diff
@@ -30,22 +30,17 @@ class MovieDetailsViewController: UIViewController {
     // MARK: - Functions
     override func viewDidLoad() {
         super.viewDidLoad()
-        updateUI()
         movieDetailsViewModel.fetchMovieDetails()
-        setupViewModelObserver()
+        movieDetailsViewModel.view = self
```

Comment:
this is very strange? why is this needed..

Reply:
it was used as part of loading the UI, will address